Package.swift

```diff
@@ -31,7 +31,7 @@ let package = Package(
         .package(url: "https://github.com/StanfordSpezi/Spezi.git", from: "1.8.2"),
         .package(url: "https://github.com/StanfordSpezi/SpeziViews.git", from: "1.10.0"),
         .package(url: "https://github.com/StanfordSpezi/SpeziStorage.git", from: "2.1.1"),
-        .package(url: "https://github.com/StanfordSpezi/SpeziNotifications.git", from: "1.0.6"),
+        .package(url: "https://github.com/StanfordSpezi/SpeziNotifications.git", branch: "lukas/fix-nextTriggerDate"),
```

Comment:
TODO: switch to tagged release once https://github.com/StanfordSpezi/SpeziNotifications/pull/13 is merged

Reply:
Just approved that 🚀